iati/iati/urls.py

```diff
@@ -43,7 +43,8 @@
     "reference",
     "rulesets",
     "schema",
-    "upgrades"
+    "upgrades",
+    "guidance/datastore/",
```

Comment:
trailing slash is not needed. You’ll end up with a double slash at the end here:

https://github.com/IATI/preview-website/blob/814787338a514447b317b2699c601641fe34f340/iati/iati/urls.py#L52